Sources/ParseSwift/ParseConstants.swift

```diff
@@ -8,14 +8,6 @@
 
 import Foundation
 
-let kParseVersion = "0.0.1"
-
-#if os(iOS)
-let kPFDeviceType = "ios"
-#elseif os(macOS)
-let kPFDeviceType = "osx"
-#elseif os(tvOS)
-let kPFDeviceType = "tvos"
-#elseif os(watchOS)
-let kParseDeviceType = "applewatch"
-#endif
+enum ParseConstants: String {
+    case parseVersion = "0.0.1"
```

Comment:
You can make this `static let parseVersion = "0.0.1"` and then you won't have to do `rawValue`. It's a small change but kinda nice.

Reply:
I didn't know this, I'll try it